File: python/Lab ปี1 ภาคต้น/week4/code/hw3.c
#include <stdio.h>

// ข้อ 3
void main(){
    int Top , Low;

    puts("Your systolic blood pressure is >> ");
    scanf("%d" , &Top);
    puts("Your lower blood pressure is >> ");
    scanf("%d" , &Low);

    if(Top < 120 && Low < 80){
        printf("Good blood pressure.");
    } else if(Top < 129 && Low < 84){
        printf("Normal blood pressure.");
    } else if(Top < 139 && Low < 89){
        printf("Quite high blood pressure.");
    } else if(Top > 139 && Low > 89){
        printf("High blood pressure.");
    }
}

File: Lab4/code/hw3.py
# ข้อ 3
Top = int(input("ความดันโลหิตตัวบนคุณคือ >> "))
Low = int(input("ความดันโลหิตตัวล่างคุณคือ >> "))

if Top < 120 and Low < 80:
    print("Good blood pressure.")
elif Top < 129 and Low < 84:
    print("Normal blood pressure.")
elif Top < 139 and Low < 89:
    print("Quite high blood pressure.")
elif Top > 139 and Low > 89:
    print("High blood pressure.")